docs/WHOLE_STUDY_IMPORT.md

```diff
@@ -0,0 +1,204 @@
+
+#### - Data clinical patient sheet
+This is the cBioPortal-formatted patient sheet that follows guidelines from [here](https://docs.cbioportal.org/5.1-data-loading/data-loading/file-formats#clinical-patient-columns)
+
+
+## Download starting file inputs 
+### cbioportal_etl/scripts/get_files_from_manifest.py
+```
+usage: get_files_from_manifest.py [-h] [-m MANIFEST] [-f FTS] [-at AWS_TBL] [-sp SBG_PROFILE] [-c CBIO] [-ao] [-rm] [-d] [-o]
+
+Get all files for a project.
+
+options:
+  -h, --help            show this help message and exit
+  -m MANIFEST, --manifest MANIFEST
+                        csv list of of genomic file location manifests
+  -f FTS, --file-types FTS
+                        csv list of workflow types to download
+  -at AWS_TBL, --aws-tbl AWS_TBL
+                        Table with bucket name and keys to subset on
+  -sp SBG_PROFILE, --sbg-profile SBG_PROFILE
+                        sbg profile name. Leave blank if using AWS instead
+  -c CBIO, --cbio CBIO  Add cbio manifest to limit downloads
+  -ao, --active-only    Set to grab only active files. Recommended.
+  -rm, --rm-na          Remove entries where file_id and s3_path are NA. Useful for studies (like pbta_all) with external files not to be downloaded when using cbio_file_name input file
+                        as manifest
+  -d, --debug           Just output manifest subset to see what would be grabbed
+  -o, --overwrite       If set, overwrite if file exists
+```
+You can run this script to verify that all required starting files have been downloaded. This also serves as an additional check to confirm that you have access to all necessary files.
+### cbioportal_etl/scripts/check_downloads.py
+```
+usage: check_downloads.py [-h] [-ms MANIFEST]
+
+Check that files were downloaded
+
+options:
+  -h, --help            show this help message and exit
+  -ms MANIFEST, --manifest-subset MANIFEST
+                        tsv list of desired genomic files
+```
+
+## Generate and validate cBiopotal load package
+After downloading the genomic files and files above as needed, this script should generate and validate the cBioPortal load package
+
+### cbioportal_etl/scripts/genomics_file_cbio_package_build.py
+```
+usage: genomics_file_cbio_package_build.py [-h] [-m MANIFEST] [-sc STUDY_CONFIG] [-dgd [{both,kf,dgd}]] [-l] [-ad]
+
+Download files (if needed), collate genomic files, organize load package.
+
+options:
+  -h, --help            show this help message and exit
+  -m MANIFEST, --manifest MANIFEST
+                        Download file manifest with cbio project, kf bs ids, cbio IDs, and file names
+  -sc STUDY_CONFIG, --study-config STUDY_CONFIG
+                        cbio study config file
+  -dgd [{both,kf,dgd}], --dgd-status [{both,kf,dgd}]
+                        Flag to determine load will have pbta/kf + dgd(both), kf/pbta only(kf), dgd-only(dgd)
+  -l, --legacy          If set, will run legacy fusion output
```

Comment:
I dropped this flag here and in all tools that reference it. We are not going back to legacy loads